OmniBLE/PumpManagerUI/ViewModels/OmniBLESettingsViewModel.swift

```diff
@@ -170,7 +184,15 @@ class OmniBLESettingsViewModel: ObservableObject {
         dateFormatter.dateStyle = .none
         return dateFormatter
     }()
-
+    
+    let timeRemainingFormatter: DateComponentsFormatter = {
+        let dateComponentsFormatter = DateComponentsFormatter()
+        dateComponentsFormatter.allowedUnits = [.hour, .minute]
```

Comment:
Since you're always overriding this in the `serviceTimeRemainingString` var,  maybe remove this line, to help avoid creating a bad assumption for the reader who may be thinking about re-using this formatter.

Reply:
As an alternative, would you accept always showing hours and minutes here, since the last 8 hours before stoppage can all be thought of as critical and we're giving up perfect consistency with the lifecycleProgress view in the conversation below? It would be a reasonable continuation of the last two hours of the 72-hour window from the progress view to just keep hours and minutes for the remainder, and then this formatter could reasonably be reused.

Reply:
Yes, I think that's even better.